cves/kernel/CVE-2014-0069.yml

```diff
@@ -324,8 +346,11 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
```

Comment:
I think you should add who exactly was the developer who signed off

Reply:
added thanks.